src/secp256k1.c

```diff
@@ -63,14 +60,18 @@ struct secp256k1_context_struct {
     secp256k1_ecmult_gen_context ecmult_gen_ctx;
     secp256k1_callback illegal_callback;
     secp256k1_callback error_callback;
+#if SECP256K1_CHECKMEM_ENABLED
     int declassify;
+#endif
 };
 
 static const secp256k1_context secp256k1_context_static_ = {
     { 0 },
     { secp256k1_default_illegal_callback_fn, 0 },
-    { secp256k1_default_error_callback_fn, 0 },
-    0
+    { secp256k1_default_error_callback_fn, 0 }
+#if SECP256K1_CHECKMEM_ENABLED
+    , 0
+#endif
```

Comment:
(mostly educational) nit. Trailing commas are allowed in initializer lists, even back in C89:  
```suggestion
    { secp256k1_default_error_callback_fn, 0 },
#if SECP256K1_CHECKMEM_ENABLED
    0,
#endif
```